lib/auth.js

```diff
@@ -32,46 +33,46 @@ async function auth() {
     return Buffer.from(`${username}:${token}`).toString('base64');
   }
 
-  // Ask the user for input and write to ~/.ncurc, then try again
-  process.stdout.write('Please enter your Github user information:\n' +
-      '[Github tokens can be created as described in ' +
-      'https://help.github.com/articles/' +
-      'creating-a-personal-access-token-for-the-command-line/]\n');
-  username = await prompt('Github user name');
-  token = await prompt('Github token');
-  check(username, token);
-  const json = JSON.stringify({ username, token }, null, '  ');
+  // Ask the user for input, create a token via github v3 API
+  // then write to ~/.ncurc and try auth() again
+  let credentials;
+  try {
+    credentials = await getCredentials({
```

Comment:
Right now this displays:

```
Reading configuration for node-core-utils failed:
ENOENT: no such file or directory, open '/Users/joyee/.ncurc'
Your GitHub username:
```

for first-time users. This looks a bit alarming, maybe pring a line saying something like

```
If this is the first time you run this command,
you can follow the instructions to create the access token.
If you prefer to create the token yourself on Github,
see <readme link>.
```